SmartDeviceLink/SDLSecondaryTransportManager.m

```diff
@@ -509,6 +509,9 @@ - (BOOL)sdl_startTCPSecondaryTransport {
     [protocol.protocolDelegateTable addObject:self];
 
     self.secondaryProtocol = protocol;
+    if(self.primaryProtocol.securityManager) {
+        self.secondaryProtocol.securityManager = self.primaryProtocol.securityManager;
```

Comment:
I think you can just remove the if statement

Reply:
done